pkgs/by-name/kr/krun/package.nix

```diff
@@ -0,0 +1,82 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, rustPlatform
+, cargo
+, libiconv
+, libkrun
+, makeWrapper
+, rustc
+, passt
+, sommelier
+, mesa
+, opengl-driver ? mesa.drivers
+, withSommelier ? false
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krun";
+  version = "0.0.0-unstable-2024-06-18";
+
+  src = fetchFromGitHub {
+    owner = "slp";
```

Comment:
https://github.com/slp/krunvm seems to be outdated (last commit on `main` 2 years ago), while forked off https://github.com/containers/krunvm, which was last touched two weeks ago (by `slp`). Are you sure the upstream is correct?

Reply:
krun != krunvm

Reply:
Ref: https://github.com/containers/krunvm/issues/62#issuecomment-2135123678

Reply:
The differences between krun and krunvm are: krunvm uses OCI images and relies on buildah to work, it runs a VM. krun is a VM sandbox of your host OS and wraps things in a nice environment for systems like Apple Silicon.